pysat/_constellation.py

```diff
@@ -455,66 +453,82 @@ def _set_inst_attr(self, attr, value):
 
     @property
     def bounds(self):
-        return self.instruments[0].bounds
+        """Obtain boundaries for Instruments in Constellation.
 
-    @bounds.setter
-    def bounds(self, value=None):
-        """ Sets boundaries for all Instruments in Constellation
+        When setting, sets for all instruments in Constellation.
 
         Parameters
         ----------
         value : tuple or NoneType
             Tuple containing starting time and ending time for Instrument
             bounds attribute or None (default=None)
-
         """
+        return self.instruments[0].bounds
 
+    @bounds.setter
+    def bounds(self, value=None):
+        # Set the bounds property.  See property docstring for details
```

Comment:
Note: when defining a getter and setter on a property, it is recommended that only the getter has the docstring, with pertinent info for setter there.  This is because only the getter docstring will be accessible via `help(pysat.Constellation.bounds)`.

Nothing specific in numpydoc about this, but examples at
https://developer.lsst.io/v/DM-15183/python/numpydoc.html#py-docstring-property-structure